ocfweb/docs/docs/staff/policies/staff-policy.md

```diff
@@ -0,0 +1,91 @@
+﻿[[!meta title="Staff policy"]]
+
+Staff members shall be chosen by the OCF Decision Making Process. [per OCF Board
+decision on 4/13/89]
+
+[The rest of this policy is per Site Manager decision on 8/30/17]
+
+## Background
+
+The OCF [Decision Making Process][0] spells out the powers of BoD, the GMs, and
```

Comment:
"OCF Decision Making Process" is a proper noun and the hyperlink should encompass the whole thing.